tests/resources/Util.robot

```diff
@@ -177,6 +177,8 @@ Run VIC Machine Delete Command
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  Completed successfully
     [Return]  ${output}
+    Run  rm -f ${vch-name}/*
```

Comment:
Prefer just a:
Run  rm -rf ${vch-name}

don't need both and whether it works or not, it should be above the return statement, actually above the should be equals, so as to limit the possibility of it not happening due to vic-machine problem.